src/app/clusters/bindings/bindings.cpp

```diff
@@ -193,19 +227,17 @@ CHIP_ERROR BindingTableAccess::WriteBindingTable(const ConcreteDataAttributePath
         {
             CreateBindingEntry(iter.GetValue(), path.mEndpointId);
         }
-        LogErrorOnFailure(NotifyBindingsChanged());
```

Comment:
This still needs to be called if `!path.IsListOperation()`, right?  Because there is no "list write end" notification in that case...

That said, it's not clear to me that we can ever actually be in this case, but then why are we checking for it above?  Please file a followup issue to get the contract here between the WriteHandler and the AttributeAccessInterface implementations pinned down, please.

Reply:
Yeah, you are right. I will add a check at the end that if this was not from a list operation it will notify anyways. I have no idea if this, as you say, can even happen. I will create an issue and we can investigate.